packages/styles/options-menu.css

```diff
@@ -18,7 +18,6 @@
 
 .OptionsMenu {
   position: relative;
-  width: 34px;
```

Comment:
This is an erroneous value and should not be there. It doesn't appear to effect other products from testing.